automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -776,3 +778,20 @@ def csv_reader(component, subcommand):
         entity_list.append(row)
     comp_dict[component] = entity_list
     return comp_dict
+
+
+def create_setup_dict(dict):
+    data.update(dict)
+    print data
+    with open('{}.pkcl'.format('product_setup'), 'wb') as pref:
```

Comment:
You can directly keep the name as 'product_setup.pkcl', need not format the string.